### DISCUSSION TITLE
[Improvement]: Metrics tab for services

### DISCUSSION THREAD

**Discussion by peaklabs-dev** [coollabsio/coolify#3153] [Improvement Requests]
### Description

The new metrics feature is amazing, but the metrics tab on the left sidebar is missing for services.

### Minimal Reproduction (if possible, example repository)

1. Enable metrics and check the sidebar on services for example WordPress there are no metrics there yet (maybe not yet implemented)

### Version

v4.0.0-beta.306

**Comment by andrasbacsai** [comment]
It is not yet implemented.

**Comment by aminhir** [comment]
no updates?
